doc/http-api/client_swagger.yml

```diff
@@ -76,30 +85,188 @@ paths:
             type: array
             items:
               $ref: '#/definitions/RecvMessage'
+  /rooms:
+    post:
+      description: Creates a room.
+      parameters:
+        - in: body
+          name: room
+          required: true
+          schema:
+            $ref: '#/definitions/CreateRoomBody'
+      responses:
+        200:
+          description: The room was successfully created
+          schema:
+            $ref: '#/definitions/ResourceID'
+  /rooms/{id}:
+    get:
+      description: "Returns room's details"
+      parameters:
+        - in: path
+          name: id
+          required: true
+          type: string
+          description: The ID of a room
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: '#/definitions/RoomDetails'
+    put:
```

Comment:
Since this endpoint is not requiring the caller to provide the whole entity that's being modified, it doesn't match with the HTTP logic behind PUT. I would recommend 2 different approaches:

1. Using `POST /rooms/{id}/invites`. This will have the additional benefit of allowing us to provide a `GET|PUT|DELETE /rooms/{id}/invites/{invite-id}` endpoint, which is cool.
2. Using `PATCH /rooms/{id}`. This will have the additional benefit of flexibility.  `PATCH` is like a wildcard in HTTP terms, you can almost do whatever you want with it.